deploy/examples/rgw-with-cephblockpool.yaml

```diff
@@ -0,0 +1,135 @@
+#################################################
+# Create an object store using pre-created pools
+#  kubectl create -f rgw-with-cephblockpool.yaml
+#################################################
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: rwc
+  namespace: rook-ceph
+spec:
+  gateway:
+    port: 80
+    instances: 1
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: rgw.root
+  namespace: rook-ceph
+spec:
+  application: rgw
+  failureDomain: osd
+  name: .rgw.root
+  parameters:
+    pg_autoscale_mode: "off"
+    pg_num: "1"
+  replicated:
+    requireSafeReplicaSize: false
+    size: 1
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: rwc.rgw.control
+  namespace: rook-ceph
+spec:
+  application: rgw
+  failureDomain: osd
```

Comment:
If the replica is changed to 3, also change this to `host`. But for a test example, this is fine.